greeting-service/.openshift/cacheserver.yaml

```diff
@@ -0,0 +1,79 @@
+# copied from https://github.com/snowdrop/spring-boot-cache-booster/blob/master/greeting-service/.openshiftio/cacheserver.yml
```

Comment:
The file needs to be in a directory called `.openshiftio` (note the _io_ at the end), and needs to be called `service.*.yaml`. I'd personally call it `service.jdg.yaml`, but `service.cacheserver.yaml` works too.

Reply:
done